packages/steps/src/BuildStepInput.ts

```diff
@@ -103,18 +108,65 @@ export class BuildStepInput<
     this.allowedValueTypeName = allowedValueTypeName;
   }
 
-  public get value(): BuildStepInputValueTypeWithRequired<T, R> {
+  public isFunctionCall(): boolean {
+    return this.rawValue?.toString().match(BUILD_STEP_FUNCTION_EXPRESSION_REGEXP) !== null;
```

Comment:
`rawValue?` means if there is no `rawValue` it will be `undefined` which will be `!== null` and will say it is a function call. Seems unexpected to me?